catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/app/UserApplication.java

```diff
@@ -152,7 +217,10 @@ private Map getSubjectPreferences(Subject subject) {
       if (preferencesList.size() == 1) {
         byte[] json = (byte[]) preferencesList.get(0).get("preferences_json_bin");
 
-        return GSON.fromJson(new String(json, Charset.defaultCharset()), MAP_STRING_TO_OBJECT_TYPE);
+        Map<String, Object> attributes =
+            GSON.fromJson(new String(json, Charset.defaultCharset()), MAP_STRING_TO_OBJECT_TYPE);
+        attributes.put("alerts", getSubjectNotifications(subject));
```

Comment:
❗️ These are good semantics for failure, if combining the data fails, the entire things fails. However it's a bit confusing that the join for retrieval happens on the backend, and the join for persistence happens on the frontend (no obvious symmetry). Can we add a comment here with the reasoning and a reference (alert-item.view) to the opposite logic on the frontend? With a mirroring comment on the frontend as well.